internal/scorecard/scorecard.go

```diff
@@ -49,57 +43,62 @@ const (
 
 // make a global logger for scorecard
 var (
-	logReadWriter io.ReadWriter
-	log           = logrus.New()
+	LogReadWriter io.ReadWriter
+	Log           = logrus.New()
```

Comment:
Why it was changed for `Log`? Will these var be used outside?

Reply:
yep, this Log was exported so that the cmd package can access it.